.yamllint

```diff
@@ -0,0 +1,8 @@
+---
+extends: default
+
+rules:
+  # 80 chars should be enough, but don't fail if a line is longer
```

Comment:
So, what is the suggestion here? 80 or 200?
I suggest to decide on one and to remove this comment.

In any case, it's good not to fail if lines are longer (as you are correctly doing), since e.g. paths to executables may be extremely long